x-pack/plugins/transform/public/app/common/pivot_aggs.ts

```diff
@@ -315,3 +315,15 @@ export function getEsAggFromAggConfig(
 
   return result;
 }
+
+export const isFilterBooleanAgg = (config: PivotAggsConfig) => {
+  if (
+    config.agg === 'filter' &&
+    isPopulatedObject(config.subAggs) &&
+    isPopulatedObject(config, ['aggConfig']) &&
+    isPopulatedObject(config.aggConfig, ['filterAgg']) &&
+    config.aggConfig.filterAgg === 'bool'
+  )
+    return true;
+  return false;
+};
```

Comment:
Interesting that the linting allows `ifs` without curly braces, I thought we had a code style recommendation to always use them 🤔 . In this case I guess you could also refactor to just:

```
export const isFilterBooleanAgg = (config: PivotAggsConfig) => (
  config.agg === 'filter' &&
  isPopulatedObject(config.subAggs) &&
  isPopulatedObject(config, ['aggConfig']) &&
  isPopulatedObject(config.aggConfig, ['filterAgg']) &&
  config.aggConfig.filterAgg === 'bool'
);
```

Reply:
Fixed here https://github.com/elastic/kibana/pull/137773/commits/1b095b0067453705d9363084d9c566c58dcff705